imsv-docs-astro/src/content/docs/guides/card-issuing-apps/creating-a-funding-channel.mdoc

```diff
@@ -0,0 +1,61 @@
+---
+title: Creating a Funding Channel
+slug: guides/creating-a-funding-channel
+---
+
+A Funding Channel is the configuration of the card funding method. Before creating
+a Funding Channel read [Funding Modes and Protocols](/guides/funding-modes-and-protocols).
+
+Choose a protocol, token, and network that suits your use case. The combination of
+these parameters dictates the [Funding Type](/guides/funding-types/) to use when
+creating a Funding Channel.
+
+## On-Chain Funding
+
```

Comment:
On-chain Funding Channel creation requires the prior deployment of a [Funding Storage Contract](/guides/funding-types/). By creating a Funding Channel you are registering this contract with Immersve. This will ensure deposits to the contract are scoped to your application.

Reply:
Does this blurb add under the On-Chain Funding? Just want to confirm the [Funding Storage Contract](/guides/funding-types/) links to `funding-types`.

Reply:
Yes under On-Chain Funding. 
Lets send the link to /guides/funding-modes-and-protocols
I will send it to the On-Chain section of that doc once it is published.